packages/openneuro-app/src/scripts/datalad/fragments/dataset-history.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from '@emotion/styled'
+import gql from 'graphql-tag'
+import { useQuery } from 'react-apollo'
+
+const GET_HISTORY = gql`
+  query getHistory($datasetId: ID!) {
+    dataset(id: $datasetId) {
+      id
+      history
+      worker
+    }
+  }
+`
+
+const DatasetHistory = ({ datasetId }) => {
+  const { loading, data } = useQuery(GET_HISTORY, {
+    variables: { datasetId },
+  })
+  if (loading) {
+    return <div className="dataset-history">Loading...</div>
+  } else {
+    return (
+      <div className="dataset-history">
+        <div className="col-xs-6">
+          <h4>Worker Assignment</h4> {data.dataset.worker}
+        </div>
+        <div className="col-xs-12">
+          <h4>Git History</h4>
+          {data.dataset.history.map(row => (
+            <div key={row}>{row}</div>
```

Comment:
I think adding a style for `margin-top: 10px` to these rows would really improve readability